app/javascript/utilities/validateAndroidNative.js

```diff
@@ -0,0 +1,7 @@
+export function isNativeAndroid() {
```

Comment:
Thanks for exporting this! Can you add a JS Doc comment to the function?

Reply:
added.